packages/components/src/components/data-list/data-list.scss

```diff
@@ -12,6 +12,11 @@
             display: flex;
             flex-direction: column;
             height: 100%;
+
+            @include mobile {
+                flex: 1 1 auto;
+                height: 100vh;
```

Comment:
I think we might need to offset the height of other elements at the top and bottom. Is it possible to fix it without 100vh?

Reply:
I think the parent element of the container which contains DataList doesn't have enough height.

Reply:
The parent's that are wrapping AutoSizer all have height as 100%